### DISCUSSION TITLE
Scale in plot_genes_per_cell_type

### DISCUSSION THREAD

**Discussion by ChristinaSteyn** [BayraktarLab/cell2location#271] [Q&A]
Hi there, 

Thank you for your assistance with Cell2location thus far! Could you please explain to me what the scale represents in the plots generated from the plot_genes_per_cell_type. I have run the scripts and gotten an output looking something like this:
![image](https://user-images.githubusercontent.com/83014373/227729957-6633e19b-b2ed-4864-ad2e-1b8368feead8.png)

I've now adjusted the scale to run from 0 to 1 but I was wondering what the values represent and why there are some negative values (are these scaled counts?). 

Thank you in advance!

**Comment by vitkl** [comment]
These are standard scanpy spatial plots - not specific to the cell2location package. Please refer to their documentation.
Here vmax option is set to a high quantile of the count distribution. When you have too few counts you get the plots like you observe.
